src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -114,15 +115,31 @@ public void uploadOutputSummary(List<CheckResult> failed, List<CheckResult> all)
     StringBuilder builder = new StringBuilder();
     builder.append("{");
 
-    String userEmail = "unknown";
-    BasicCommand getGitUser = new BasicCommand("git", "config", "--global", "user.email");
-    getGitUser.run();
-    if (getGitUser.exitValue() == Command.SUCCESS) {
-      userEmail = getGitUser.getOutput().trim();
+    String userId = "unknown";
+
+    if (GatorGradleConfig.get().hasIdCommand() == true) {
+      BasicCommand getUserId = null;
+      if (System.getProperty("os.name").toLowerCase(Locale.ENGLISH).equals("windows")) {
+        getUserId = new BasicCommand(
+            "sh", "/C", GatorGradleConfig.get().getIdCommand());
+      } else {
+        getUserId = new BasicCommand(
+            "sh", "-c", GatorGradleConfig.get().getIdCommand());
+      }
+      getUserId.run();
+      if (getUserId.exitValue() == Command.SUCCESS) {
+        userId = getUserId.getOutput().trim();
+      }
+    } else {
+      BasicCommand getGitUser = new BasicCommand("git", "config", "--global", "user.email");
```

Comment:
Make this default go through the same process as the others -- have the default value in the config, and then parse it just the same here. I.E., have the default be returned from `hasIdCommand()` so that there's no extra code here.

Reply:
Then maybe we have a test for failure (an else in the `SUCCESS` test above) and use the hostname as a last resort or something like that, that we can guarantee we can get.

Reply:
Fixed in https://github.com/GatorEducator/gatorgradle/pull/71/commits/cc962ac0e513ad600a4639e00878f3c4922b5009 and https://github.com/GatorEducator/gatorgradle/pull/71/commits/00fa06dbf4aabc8401d8268fb0613143d71c69c7